Extractor/SessionManager.cs

```diff
@@ -544,7 +525,7 @@ private async Task UpdateServiceLevelInner(byte newLevel, bool fromConnectionCha
                 // We don't want to reconnect too frequently
                 if (lastLowSLConnectAttempt != null)
                 {
-                    var timeSinceLastAttempt = DateTime.UtcNow - lastLowSLConnectAttempt.Value;
+                    var timeSinceLastAttempt = (DateTime.UtcNow - lastLowSLConnectAttempt.Value) + TimeSpan.FromSeconds(5);
```

Comment:
Would this not have the effect of internally reducing config.Redundancy.ReconnectIntervalValue.Value by 5 seconds?